client/src/assets/icons/experimental.svg

```diff
@@ -1,2 +1,2 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-  <polygon points="8.375 4.75 7.625 4.75 7.625 6.737 6.21 9.25 9.79 9.25 8.375 6.737 8.375 4.75" fill="#3b8ac9"/><path d="M8,0a8,8,0,1,0,8,8A8,8,0,0,0,8,0Zm4.25,11.5a.75.75,0,0,1-.75.75h-7a.75.75,0,0,1-.75-.75v-.9375L6.125,6.3438V4.75H5.5V3.25h5v1.5H9.875V6.3438l2.375,4.2187Z" fill="currentColor"/></svg>
+<svg class="icon" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="img"><path d="M90.72 82.34c4.4 7 1.29 12.66-7 12.66H16.25C8 95 4.88 89.31 9.28 82.34l29.47-46.46V12.5H35A3.75 3.75 0 0135 5h30a3.75 3.75 0 010 7.5h-3.75v23.38zM45.08 39.86L29.14 65h41.72L54.92 39.86l-1.17-1.81V12.5h-7.5v25.55z" fill="currentColor"/></svg>
```

Comment:
I would remove the `aria-hidden` as this icon is often used in scenarios where the explanation for the icon is shown on hover/focus. Currently we are not doing the best job of ensuring these are focusable but, not having the icon set to hidden would be a good first step.